### PR TITLE
adapt Jina Embeddings to new Jina AI Embedding API

### PR DESCRIPTION
**Pull request by JoanFM**
- **Description:** Adapt JinaEmbeddings to run with the new Jina AI Embedding platform
- **Twitter handle:** https://twitter.com/JinaAI_

### CODE REVIEW

File: libs/langchain/langchain/embeddings/jina.py [hunk 1]
```diff
@@ -7,69 +6,43 @@
...
+    jina_api_key: Optional[str] = None
 
     @root_validator()
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that auth token exists in environment."""
         # Set Auth
-        jina_auth_token = get_from_dict_or_env(
-            values, "jina_auth_token", "JINA_AUTH_TOKEN"
+        jina_api_key = get_from_dict_or_env(
+            values, "jina_api_key", "JINA_API_KEY"
```
Comment (baskaryan) [inline libs/langchain/langchain/embeddings/jina.py:24]: for backwards compatibility should we check for JINA_AUTH_TOKEN as well if JINA_API_KEY isn't found?

Comment (JoanFM) [reply libs/langchain/langchain/embeddings/jina.py:24]: good idea, I will do

Comment (JoanFM) [reply libs/langchain/langchain/embeddings/jina.py]: I have fixed it @baskaryan, may you check again?

**Comment by JoanFM** [comment]
Hello @baskaryan , any news about this?

**Comment by meal** [comment]
Would love to see this in the next release!

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**